docs/content/src/learn/tutorials/saas-integrations/gmail/using-the-gmail-connector/README.md

```diff
@@ -1,138 +1,125 @@
 
+```bash
+$ java -jar target/bin/gmail_client_application.jar --b7a.config.file=path/to/ballerina.conf/file
 ```
-$ ballerina run health_care_service.bal
+Now we can see that the service has started on port 9090. Let’s invoke this service by executing the following cURL command.
 ```
-
-The 'healthCareService' service will start on port 9092. Now we can send an HTTP request to this service.
-
-Let's create a file called _request.json_ and add the following content.
-
-```json
-{
-  "patient": {
-    "name": "John Doe",
-    "dob": "1940-03-19",
-    "ssn": "234-23-525",
-    "address": "California",
-    "phone": "8770586755",
-    "email": "johndoe@gmail.com"
-  },
-  "doctor": "thomas collins",
-  "hospital": "grand oak community hospital",
-  "appointment_date": "2025-04-02"
-}
+curl -X GET http://localhost:9090/gmail/reviews
 ```
+You will see the list of email body contents during a successful invocation.
 
-Navigate to _using-the-gmail-connector/src/tutorial_ and send the request message to the service using cURL.
+### Writing unit tests
 
-```
-$ curl -v -X POST --data @request.json http://localhost:9092/hospitalMgtService/surgery/reserve --header "Content-Type:application/json"
-```
+In Ballerina, the unit test cases should be in the same package inside a folder named `tests`. When writing test functions the convention given below should be followed.
 
-A request is made to the _healthCareService_ which returns a JSON payload with the confirmation details such as the appointment number and fees.
-
-```json
-{
-  "appointmentNumber": 1,
-  "doctor": {
-    "name": "thomas collins",
-    "hospital": "grand oak community hospital",
-    "category": "surgery",
-    "availability": "9.00 a.m - 11.00 a.m",
-    "fee": 7000
-  },
-  "patient": {
-    "name": "John Doe",
-    "dob": "1940-03-19",
-    "ssn": "234-23-525",
-    "address": "California",
-    "phone": "8770586755",
-    "email": "johndoe@gmail.com"
-  },
-  "fee": 7000,
-  "confirmed": false,
-  "appointmentDate": "2025-04-02"
+Test functions should be annotated with `@test:Config {}`. See the example below.
+```ballerina
+@test:Config {}
+function testSendTextMessage() {
 }
 ```
 
-This payload is used to extract necessary details for the email message using the _generateEmail_ function. With the necessary _MessageRequest_ details specified, the email can be sent to its intended recipient and a response is sent back to the caller confirming the receipt of the email.
+This guide contains unit tests for the Gmail application in the `gmail_client_test.bal`.
 
-```json
-{
-  "Message": "The email has been successfully sent",
-  "Recipient": "someone@gmail.com"
-}
+To run the unit tests, navigate to the project directory and run the following command.
+```
+ballerina test
 ```
+> **Note:** The `--b7a.config.file=path/to/ballerina.conf/file` option is required if it is needed to read configurations from a Ballerina configuration file.
```

Comment:
we only need to give this if `ballerina.conf` file is not in the project root directory.

Reply:
Fixed in https://github.com/wso2/ballerina-integrator/pull/416